pom.xml

```diff
@@ -60,21 +60,34 @@
         <module>partitioner</module>
         <module>format</module>
         <module>hive</module>
+        <module>htrace-core4-shaded</module>
         <module>package-kafka-connect-storage-common</module>
     </modules>
 
     <properties>
+        <avro.version>1.10.2</avro.version>
         <commons-io.version>2.4</commons-io.version>
+        <commons.beanutils.version>1.9.4</commons.beanutils.version>
+        <commons.codec.version>1.15</commons.codec.version>
+        <confluent.log4j.version>1.2.17-cp2</confluent.log4j.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <confluent.version>5.2.5-SNAPSHOT</confluent.version>
-        <hadoop.version>2.7.3</hadoop.version>
+        <hadoop.version>2.10.1</hadoop.version>
         <hive.version>1.2.2</hive.version>
-        <joda.version>2.9.6</joda.version>
-        <licenses.version>5.2.5-SNAPSHOT</licenses.version>
-        <parquet.version>1.8.2</parquet.version>
         <httpclient.version>4.5.13</httpclient.version>
         <httpcore.version>4.4.4</httpcore.version>
+        <jackson.version>2.10.5</jackson.version>
+        <jackson.databind.version>2.10.5.1</jackson.databind.version>
+        <jackson.mapper.asl.version>1.9.13</jackson.mapper.asl.version>
```

Comment:
I'm guessing there's no universal version for these 3 modules that addresses all the CVEs

Reply:
correct